db/migrate/20220914131056_add_users_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :users, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Please edit this migration because now it's creating a foreign key as `user_id` and It's supposed to be `author_id` like the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
[Hint]: You can do it like that:
```ruby
add_reference :posts, :author, null: false, foreign_key: { to_table: 'users' }
```

Reply:
Hi @Omar-Muhamad 
Thanks for your review
I have one question: to make the changes, should I generate a new migration?
for example, should I run: rails generate migration AddUsersRefToPosts users:references
And then with the file generated I should add that, I am right?
Thanks in advance

Reply:
Thanks for your help @Omar-Muhamad 
Happy coding!